README.md

```diff
@@ -34,7 +34,7 @@ From the victim's perspective, the attack makes use in three phases:
 
 ## Requirements
 * Kali Linux.
-* Two wireless network interfaces, one capable of injection.
+* Two wireless network adapters; one capable of injection. This allows you to connect and confirm the state of the target AP.
```

Comment:
I don't think the last statement is correct. The one adapter is used for the injection, the other one for creating the fake AP. Although, I believe there's no need for an explanation in the 'Requirements' section. You can just remove the last sentence.